backend/src/main/kotlin/com/hiper2d/model/Player.kt

```diff
@@ -0,0 +1,14 @@
+package com.hiper2d.model
+
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document(collection = "player")
+data class Player(
+    @Id val id: String? = null,
```

Comment:
Do we need explicitly put "= null" ?

Reply:
When we create a player it has no id. An id appears after saving to the database. WIthout default null value we'll have to pass this null in a constructor